controllers/tasks.js

```diff
@@ -130,8 +130,19 @@ const fetchPaginatedTasks = async (query) => {
 
 const fetchTasks = async (req, res) => {
   try {
-    const { dev, status, page, size, prev, next, q: queryString, assignee, title } = req.query;
-    const transformedQuery = transformQuery(dev, status, size, page, assignee, title);
+    const {
+      dev,
+      status,
+      page,
+      size,
+      prev,
+      next,
+      q: queryString,
+      assignee,
+      title,
+      "assignee-role": assigneeRole,
+    } = req.query;
+    const transformedQuery = transformQuery(dev, status, size, page, assignee, title, assigneeRole);
```

Comment:
Nit: Should we consider passing the req.query object to transformQuery function.

Reply:
removed it as it is not being used in transformQuery function